llvm/lib/Target/AMDGPU/SIFrameLowering.cpp

```diff
@@ -1555,6 +1555,17 @@ void SIFrameLowering::determinePrologEpilogSGPRSaves(
   }
 }
 
+// Mark all WWM VGPRs as BB LiveIns.
+static void addWwmRegBBLiveIn(MachineFunction &MF) {
```

Comment:
If the registers are reserved, they don't need to be made live in. It's one or the other

Reply:
We shift back the wwm-regs to the lowest range at this point. They aren't explicitly marked reserved. I'll do that by calling MRI.reserveReg() on these new regs. With that, I can stop adding them to BBLiveIns.